### DISCUSSION TITLE
Task Filtering Shortcuts in Task List

### DISCUSSION THREAD

**Discussion by Kishub** [madrecha/jamku#780] [QA] [answered]
Hi, with respect to shortcuts used for task filtering would like to know in more detailed about the same functions used for the ease of practice instead of typing the task name, number to enhance my search option at the ease.

**Comment by adarshmadrecha** [comment, marked as answer]
Here's the exhaustive List. It's also documented in [help docs for Task List](https://help.jamku.app/guide/task-list.html#task-filtering).

Watch the video on task filtering
[<img alt="Task List filter in Jamku" width="300px" src="https://user-images.githubusercontent.com/11911938/136877094-cb848108-eb0b-4b61-a496-f0f271b7ce6e.png" />](https://youtu.be/hBvwS0X3lY8)


## Count of team members assigned to task.
* `@0` -- Task with NO Users assigned
* `@1` -- Task with ONE user assigned
* `@n` -- Task with "n" user assigned

## Period/Year
* `sep.` -- Task where the period is `sep` (September). Note the fullstop (aka period `.`) at the end of the `sep`
* `q1.` -- Task where the period is `Quarter 1` (In case of quarterly tasks)
* `h1.` -- Task where the period is `1st Half` of the year (In case of Half yearly tasks) 
* `2020.` -- Task where year is `2020`. (Irrespective of periodicity)

## Date
To filter task based on dates add the following prefix
`A:` -- Task create date (Think of it as "Added" Date, hence the `A`)
`D:` -- Task due date
`T:` -- Task target date
`C:` -- Task completion date
`V:` -- Task verified date

After choosing the filter of your choice one of the following suffixes (relative to the system date of the device)
`Yesterday` , `Today`, `Tomorrow` -- Basis single day 
`LastWeek`, `ThisWeek`, `NextWeek` -- Basis week
`ThisMonth`, `LastMonth`, `NextMonth` -- Basis Month 

_Example_:
`D:Today` -- Tasks which have due date of today
`C:ThisWeek` -- Tasks which are completed in this week (Monday to Sunday)

## Task Status
`G.` `going.` -- Taks with "Going" status
`S.` `stuck.` -- Taks with "Stuck" status
`C.` `completed.` -- Taks with "Completed" status
`I.` `ignore.` -- Taks with "Ignore" status 

## Task Priority
<img src="https://user-images.githubusercontent.com/11911938/136878046-65663920-2ced-449f-b592-ab0a90af1acf.png" width="450px" alt="Task Priority in Jamku. Task Edit Dialogue Box" />

`P1.` -- Tasks with "Critical" Priority
`P2.` -- Tasks with "High" Priority
`P3.` -- Tasks with "Normal" Priority
`P4.` -- Tasks with "Low" Priority
`P5.` -- Tasks with "Least" Priority
